tests/unit/test_models/test_full_battery_models/test_lithium_ion/test_spm_half_cell.py

```diff
@@ -0,0 +1,58 @@
+#
+# Tests for the lithium-ion half-cell SPM model
+# This is achieved by using the {"working electrode": "positive"} option
+#
+import pybamm
+import unittest
+
+
+class TestSPMHalfCell(unittest.TestCase):
```

Comment:
Here they are called `TestSPMHalfCell` as opposed to `TestSPM` as before. It would be good to unify the notation.